test/fixtures/hoc/actual.js

```diff
@@ -0,0 +1,11 @@
+class Component extends React.Component {
+  render() { return null }
+}
+
+Component.propTypes = {
+  children: React.PropTypes.string.isRequired,
+  onClick: React.PropTypes.func,
+  style: React.PropTypes.object,
+}
+
+export default withHoc()(deeperHoc(Component))
```

Comment:
would you mind adding another test with just 1 hoc? it's a slightly different test case and I don't want this to regress in case someone refactors this.